### DISCUSSION TITLE
Image Update Automation Push to Prefixed Branch

### DISCUSSION THREAD

**Discussion by alfkonee** [fluxcd/flux2#4439] [Q&A]
Hello there, 
I was wondering if it's possible to to push an Image Update to a prefixed or suffixed branch  using a template like we have in the message. I think it'll be a very nice feature to help with the Auto PR Flow using the gh cli or gl cli  or any other  repo tool's cli that allows for pr creation from the cli

**Comment by makkes** [comment]
Right now you can only use a fixed branch name as described [in the docs](https://fluxcd.io/flux/components/image/imageupdateautomations/#push). I'm curious to understand why you would need different branches, though. Usually you would configure IAU to push to the same branch every time so that you don't end up with different PRs for the same changes.
